modules/hal_nordic/nrf_802154/sl_opensource/platform/nrf_802154_temperature_zephyr.c

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ *   This file implements the thermometer abstraction that uses Zephyr sensor
+ *   API for the die termomether.
+ *
+ */
+
+#include "platform/nrf_802154_temperature.h"
+
+#include <device.h>
+#include <drivers/sensor.h>
+#include <kernel.h>
+#include <stdbool.h>
+
+#define TEMPERATURE_UPDATE_THREAD_STACK_SIZE	128
+#define TEMPERATURE_UPDATE_THREAD_PRIORITY	5
+
+/** @brief Thread handler for temperature update. */
+static void temperature_update_thread(void *p1, void *p2, void *p3);
+
+K_THREAD_DEFINE(temperature_update_tid,
```

Comment:
can be defined at the end to avoid the previous declaration

Reply:
fixed